setup-tests.ps1

```diff
@@ -0,0 +1,43 @@
+# Check if we're running on Windows, otherwise return as we have nothing to do on other platforms
+if ($IsWindows -eq $false) {
+  return
+}
+
+# Making sure we have the scripts current folder
+$root = $PSScriptRoot
+
+if ("$root" -eq "") {
```

Comment:
I'm fairly sure you don't need to quote variables in Powershell, because it doesn't do word splitting on expansion, unlike bash. I'm not doing it in my own Powershell scripts and they work without problems on stuff like paths with whitespace. You only need to quote when you want to make sure that the variable is turned into a string or when you want to write a literal with white space (with or without variables in it). I also usually add them to string literals in general to be more clear, but you don't even need to do that.